src/scripts/build/nimbusTypes.js

```diff
@@ -91,7 +91,19 @@ function getFallbackObject(nimbusConfig) {
     },
   );
 
-  return `export const defaultExperimentData: ExperimentData = {\n${featureFallbackDefs.join("\n")}};\n`;
+  const defaultExperimentData = `
+    "Features": {
+      ${featureFallbackDefs.join("\n")}
+    },
+    "Enrollments": {
+      "nimbus_user_id": "-1",
+      "app_id": "-1",
+      "experiment": "-1",
+      "branch": "-1",
+      "experiment_type": "-1",
+      "is_preview": false
+    }`;
```

Comment:
The newly added string literal makes the resulting `experiment.ts` harder to grasp. Would be nice if we could fix the formatting for `"Features"` and `"Enrollments"`. Here is an example of the current output:

```
export const defaultExperimentData: ExperimentData = {

    "Features": {
        "last-scan-date": {
    "enabled": false,
  },

  "welcome-scan-optional-info": {
    "enabled": false,
    "variant": "none",
  },

  "data-privacy-petition-banner": {
    "enabled": false,
  },

  "landing-page-free-scan-cta": {
    "enabled": false,
    "variant": "ctaWithEmail",
  },

  "data-broker-removal-time-estimates": {
    "enabled": false,
  },

    },
    "Enrollments": {
      "nimbus_user_id": "-1",
      "app_id": "-1",
      "experiment": "-1",
      "branch": "-1",
      "experiment_type": "-1",
      "is_preview": false
    }};
```

Reply:
Thanks, yeah I think we should use some smarter tooling for code generation... for now I can fix the formatting by hand.

Reply:
Looked into this further - we can actually just not worry about formatting on the generation side and call `prettier` programmatically when writing the file.